frb/data/Galaxies/20200906A/FRB20200906A_host.json

```diff
@@ -136,21 +210,23 @@
         "VLT_FORS2_g_flux": 0.021604857893556356,
         "VLT_FORS2_g_flux_err": 0.0010182055833693284,
         "VLT_FORS2_g_ref": "Bhandari2021",
-        "WISE_W1": 19.352344512939453,
-        "WISE_W1_err": 0.1509999930858612,
-        "WISE_W1_flux": 0.06592684984207153,
-        "WISE_W1_flux_err": 0.009837053716182709,
-        "WISE_W2": 19.364715576171875,
-        "WISE_W2_err": 0.2329999953508377,
-        "WISE_W2_flux": 0.06517989933490753,
-        "WISE_W2_flux_err": 0.01560193207114935,
-        "WISE_W3": 16.824451446533203,
+        "WISE_W1": 19.63907182486113,
+        "WISE_W1_err": 0.151,
+        "WISE_W1_flux": 0.05062572600783015,
+        "WISE_W1_flux_err": 0.007553945437201975,
+        "WISE_W1_ref": "Gordon2023",
+        "WISE_W2": 19.71012496951422,
+        "WISE_W2_err": 0.233,
+        "WISE_W2_flux": 0.04741873963614091,
+        "WISE_W2_flux_err": 0.011350491162822685,
+        "WISE_W2_ref": "Gordon2023",
+        "WISE_W3": 16.824386596679688,
         "WISE_W3_err": -999.0,
-        "WISE_W3_flux": 0.676424503326416,
+        "WISE_W3_flux": 0.6764653325080872,
         "WISE_W3_flux_err": -99.0,
-        "WISE_W4": 14.974725723266602,
+        "WISE_W4": 14.974658966064453,
         "WISE_W4_err": -999.0,
```

Comment:
999 or -999?

Reply:
Should be 999. New WISE update should fix it - will rerun